flink-kubernetes/src/main/java/org/apache/flink/kubernetes/KubernetesClusterClientFactory.java

```diff
@@ -66,6 +67,7 @@ public String getClusterId(Configuration configuration) {
 	}
 
 	private String generateClusterId() {
-		return CLUSTER_ID_PREFIX + UUID.randomUUID();
+		final String randomUUID = UUID.randomUUID().toString().replaceAll("-", "");
```

Comment:
Instead of doing the string replacement, we could also use
```suggestion
		final String randomID = new AbstractID().toString();
```

this would be a bit simpler.

Reply:
> Instead of doing the string replacement, we could also use
> 
> this would be a bit simpler.

I agree that the suggested solution would be a bit simpler. The newest commit has adopted it.